### PR TITLE
Icclim prcptot

### LINKED ISSUE #812: Possible modification to PRCPTOT
* xclim version: 0.27.0
* Python version: 3.8 - running in Jupyter
* Operating System: Linux

### Description
I would like to suggest a modification to the ```xclim.indicators.atmos.precip_accumulation()``` function for PRCPTOT

Currently the function does not match the definition of PRCPTOT given by the ETCCDI for annual precipitation and extreme indices. The modification will need to allow a precipitation threshold similar to the one in the ```xclim.indicators.icclim.SDII()``` function.

The definition of PRCPTOT for extreme precipitation is:

RCPTOT | Annual total wet-day PR | Sum of daily PR >= 1.0 mm | Total wet-day rainfall | mm | 

I have done an evaluation looking at the mean of the differences over 20 years for R95pTOT, R99pTOT, RX1day, RX5day, SDII, R10mm, R20mm, CDD and CWD and they all return a nearly identical match. Hence, this will be a great improvement for xclim particularly when looking at extreme precipitation indices. . 

The current work around is to simply modify the file containing the data using CDO, by changing values <= 1.0 mm to 0 mm

Thanks.

### PR DESCRIPTION
**Pull request by huard**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #812 
- [x] Tests for the changes have been added (for bug fixes / features)
       Maybe a bit weak on tests. Hopefully someone can jump in. 
- [ ] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
- [ ] `bumpversion (major / minor / patch)` has been called on this branch
- [ ] Tags have been pushed (`git push --tags`)
- [ ] The relevant author information has been added to `.zenodo.json`

### What kind of change does this PR introduce?

* ,,,

### CODE REVIEW

File: xclim/data/cf.yml [hunk 1]
```diff
@@ -3104,44 +3104,38 @@ indices:
...
+      standard_name: lwe_thickness_of_precipitation_amount
+      long_name: "Total precipitation during Wet Days"
+      units: "mm"
+      cell_methods:
+        - time: sum within days
+        - time: mean over days
+    input:
+      data: pr
+    index_function:
+      name: wet_precip_accumulation
```
Comment (aulemahal) [inline xclim/data/cf.yml:3126]: I understand what you are trying to do, but I would really like if integration of the clix-meta indicators was unassisted, to lower maintenance.
Let's leave this, but the rate vs amount think needs to be sorted out upstream.

File: xclim/data/fr.json [hunk 2]
```diff
@@ -201,6 +201,13 @@
     "comment": "",
     "abstract": "Précipitation accumulée totale. Si la température journalière moyenne est donnée, le paramètre phase peut-être utilisé pour restreindre le calcul aux précipitations d'une seule phase (liquide ou solide). Les précipitations sont considérées solides si la température journalière moyenne est sous 0°C (et vice-versa)."
   },
+  "WET_PRCPTOT": {
+    "long_name": "Précipitation totale lors de jours pluvieux",
+    "description": "Précipitation totale {freq:f} lorsque la précipitation journalière excède {thresh}.",
+    "title": "Précipitation accumulée totale lors de jours pluvieux",
+    "comment": "",
+    "abstract": "Précipitation accumulée totale lors de jours pluvieux. Un jour est considéré pluvieux si la précipitation est supérieure ou égale à {thresh}."
```
Comment (aulemahal) [inline xclim/data/fr.json:209]: Can't have format strings in "abstract" or "title".
```suggestion
    "abstract": "Précipitation accumulée totale lors de jours pluvieux. Un jour est considéré pluvieux si la précipitation est supérieure ou égale à un seuil donné."
```

**Review by aulemahal — APPROVED** [review]
A few comments on the metadata. 

I'm not a big fan of the `precip_accumulation` vs `prcptot` situation, but this is not such an issue for now. Let's wait until someones wants `wet_liquid_prcptot` before rethinking it.

**Review by Zeitsperre — APPROVED** [review]
LGTM

File: xclim/data/cf.yml (hunk 1 above)
Comment (huard) [reply xclim/data/cf.yml:3126]: Agreed.

**Comment by huard** [comment]
@aulemahal Any idea why an sdba test would fail ?

**Comment by aulemahal** [comment]
Scheisse. Looks like #831 ...

**Pull request merged**